portfolio.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Laura's portfolio</title>
+
+  <link href="styles/normalize.css" rel="stylesheet">
+  <link href="styles/portfolio_style.css" rel="stylesheet">
+</head>
+<body>
+  <header class="clearfix">
+    <h1> Laura's Portfolio</h1>
+    <nav>
+      <ul>
+        <li><a href="index.html"> Home </a></li>
+        <li><a href="portfolio.html"> Portfolio </a></li>
+        <li><a href="journal.html"> Journal </a></li>
+        <li><a href="About.html"> About </a></li>
+      </ul>
+    </nav>
+  </header>
+  <main>
+    <h2> PORTFOLIO</h2>
+
+    <articule class="clearfix">
+        <h3> RIDE SHARE TWO </h3>
+        <div class="right">
+        <p> This project manages data related to rides. Kepping track of drivers, rides and trips that they have taken.
+          You can ask for a certain driver, rider or trip, and the info about each one will be shown to you using their ID's for example.
+          There are more things to learn about this project! clikc <a href="https://github.com/lmelgarejos/ride-share-two">here</a> and find out more info.
+        </p>
+      </div>
+      <div class="left">
+        <img src="http://www.cooneyconway.com/sites/default/files/5-rideshare-definition-connect-potential-drivers-with-passengers.png" alt="ride_share">
+      </div>
+    </articule>
+
+    <articule class="clearfix">
+      <h3> SCRABBLE GAME </h3>
+      <div class="left">
+        <p> This project keeps track of most aspects of the game scrabble, like the words that a player has made, the scores, and new random letters
+          to keep having fun!
+          Click <a href="https://github.com/lmelgarejos/Scrabble">here</a> to find more info.
+        </p>
+      </div>
+      <div class="right">
+        <img id="img_scrabble" src="https://s-media-cache-ak0.pinimg.com/originals/3a/86/2b/3a862bc24dc905a9312a5ac2749f6d1e.jpg" alt="ride_share">
+      </div>
+    </articule>
+
+    <articule class="clearfix">
+      <h3> SOLAR SYSTEM </h3>
+
+      <div class="right">
+        <p> This project will provide you with facts about one specific planet in our solar system. You need to write the name of the planet which you want the basic information about,
+          and the program will display the facts about the planet. Click <a href="https://github.com/lmelgarejos/Solar-System">here</a> to find more info.
+        </div>
+        <div class="left">
+          <img src="http://www.seasky.org/solar-system/assets/animations/solar_system_menu.jpg" alt="ride_share">
+        </div>
+      </articule>
+    </main>
+
```

Comment:
This main indent is a little off from the opening main and extra whitespace here.